### PR TITLE
[HDRP]Fixed parameters order on inspectors for Volume Components without custom editor

### PR DESCRIPTION
**Pull request by alex-vazquez**
### Purpose of this PR
This PR is fixing this
- https://fogbugz.unity3d.com/f/cases/1294683/
#### Before PR
image
#### After PR:
image
---
### Testing status
- Check that the `VolumeParameters` are being displayed in the same way as they are declared
---
### Comments to reviewers
This PR is simply backporting the `VolumeComponentInspector.cs` file from https://github.com/Unity-Technologies/Graphics/pull/2745

### CODE REVIEW

**Review by sebastienlagarde — APPROVED** [review]

**Review by jenniferd-unity — APPROVED** [review]

File: com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs [hunk 1]
```diff
@@ -202,37 +212,27 @@ public override int Compare((GUIContent displayName, int displayOrder, Serialize
...
+                        name = attr[0].name;
+                        order = attr[0].order;
+                    }
 
-                var parameter = new SerializedDataParameter(property);
-                m_Parameters.Add((new GUIContent(name), order, parameter));
-            }
-            m_Parameters.Sort(new ParameterSorter());
+                    var parameter = new SerializedDataParameter(t.Item2);
+                    return (new GUIContent(name), order, parameter);
```
Comment (RSlysz) [inline com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs:232]: If I get it well, here will be the field name right? Shouldn't it be localizable?

**Review by RSlysz — APPROVED** [review]
Pre approved but check localization.

File: com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs (hunk 1 above)
Comment (alex-vazquez) [reply com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs:232]: yeah, that's true, I will create a another task to improve this. Thanks Rémi

**Review by TomasKiniulis — APPROVED** [review]
Checked fogbugz case custom volume, also created a new one and made sure that existing Volume overrides are not affected. Looks good! Order is correct now in all volumes

**Review by martint-unity — APPROVED** [review]

**Pull request merged**